code/graph_algorithms/data_structures/adjacency_matrix_C/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Part of Cosmos by OpenGenus Foundation.
+ * Author : ABDOUS Kamel
+ * How to use matrix-representation graphs.
+ */
+
+#include "mgraph_struct.h"
+#include <stdio.h>
+
+#define NB_NODES 5
+
+int
+main()
+{
+  M_Graph g;
```

Comment:
I’m not entirely sure on this, but it seems you are using the wrong naming style for everything. Use camel case for variable and function names. Use pascal case for struct names.

Reply:
When i write it MGraph, it seems too contiguous, so i wrote it M_Graph.

Reply:
Ok

Reply:
Should i edit naming style ?

Reply:
It is fine :+1: